test/avocado/selenium-machines-vcpu.py

```diff
@@ -75,3 +77,18 @@ def testVcpuConfig0(self):
 
     def testVcpuConfig1(self):
         self.vcpuConfigureAndCheck('running', '16', '8', '2', '4', '2')
+
+    def testVcpuUsageOff(self):
+        self.create_vm(state='shut off')
+
+        self.click(self.wait_css('#vm-staticvm-usage', cond=clickable))
+        self.assertEqual(self.wait_css('#chart-donut-1 .donut-title-big-pf', cond=visible).text, '0.0')
+
+    @skipIf('can not get the cpu usage')
+    def testVcpuUsageRunning(self):
+        self.create_vm()
+
+        self.click(self.wait_css('#vm-staticvm-usage', cond=clickable))
+        # # wait for appearing the data
+        sleep(15)
```

Comment:
`sleep` with hardcoded numbers is not good practice. Please keep in mind that we run the selenium tests gating each PR, thus adding this 15 sec extra time to the tests in not desirable.